CMakeLists.txt

```diff
@@ -5,8 +5,10 @@ project(alicia-server
 option(BUILD_TESTS "Build tests" ON)
 
 find_package(Boost REQUIRED)
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(LIBPQXX REQUIRED libpqxx)
 
-add_subdirectory(3rd-party)
+add_subdirectory(3rd-party/spdlog)
```

Comment:
This was fine like it was before. 
`3rd-party` folder contains CMakeLists.txt file which points to subdirectories.